webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -35,7 +49,39 @@ export default {
   props: {
     placement: { type: String },
   },
+  created() {
+    setInterval(this.updateNotifications, REFRESH_MILLISEC)
+  },
+  destroyed() {
+    clearInterval(this.updateNotifications)
+  },
```

Comment:
![scared](https://media.giphy.com/media/YQA1cC40JlhB2MScI8/giphy.gif)

Please use `pollInterval` by `vue-apollo` and don't re-invent the wheel! See: https://vue-apollo.netlify.com/api/smart-query.html#options

Reply:
Yeah !!!